server/setup.py

```diff
@@ -0,0 +1,63 @@
+from pathlib import Path
+
+from setuptools import Command, find_packages, setup
+
+
+class GenerateCommand(Command):
+    description = "generates manticore_server server protobuf + grpc code from protobuf specification file"
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        from grpc.tools import protoc
+
+        protoc.main(
+            [
+                "grpc_tools.protoc",
+                "-I.",
+                "--python_out=.",
+                "--grpc_python_out=.",
+                "--mypy_out=.",
+                "./manticore_server/ManticoreServer.proto",
+            ]
+        )
+
+
+setup(
+    name="manticore_server",
+    version="0.0.1",
+    packages=find_packages(exclude=["tests", "tests.*"]),
+    python_requires=">=3.7",
+    install_requires=[
+        f"manticore[native] @ file://{Path(__file__).parent.resolve()}/..",
+        "protobuf~=3.20",
+        "grpcio~=1.46",
+        "crytic-compile>=0.2.2",
+    ],
```

Comment:
`python3 setup.py install` raises the following error: `must be a list of strings containing valid project/version requirement specifiers; Invalid URL given`

I'm guessing something is wrong with `f"manticore[native] @ file://{Path(__file__).parent.resolve()}/.."`

Reply:
I think this might be an issue with outdated `setuptools`, `pip`, and/or `wheel`.

I'm pretty sure there's a way to specify a minimum version of those but I'll need to look into it.

Did you by any chance encounter this after running `just init` or was this installation all done manually?

Reply:
I didn't use the `just` file and tried a manual installation

Reply:
Okay, I fixed this now. We require `setuptools>=49.0.0`